folio-integration-kong/src/main/java/org/folio/tools/kong/service/KongGatewayService.java

```diff
@@ -119,6 +119,7 @@ public void deleteService(String serviceNameOrId) {
     try {
       kongAdminClient.deleteService(serviceNameOrId);
     } catch (Exception e) {
+      log.warn("Failed to delete Kong service: {}", serviceNameOrId, e);
       var parameters = List.of(new Parameter().key("cause").value(e.getMessage()));
       throw new KongIntegrationException("Failed to delete Kong service: " + serviceNameOrId, parameters);
```

Comment:
Should we also pass `e` exception here as a cause?

Reply:
That is a good catch, but Parameters can be shown for API clients, so it's not a good case to have stack trace in response